src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -59,7 +59,13 @@ public override void VisitField(IFieldSymbol symbol)
                 AddPunctuation(SyntaxKind.DotToken);
             }
 
-            if (symbol.ContainingType.TypeKind == TypeKind.Enum)
+            if (symbol.AssociatedSymbol is IPropertySymbol associatedProperty)
+            {
+                AddPropertyNameAndParameters(associatedProperty);
+                AddPunctuation(SyntaxKind.DotToken);
+                AddKeyword(SyntaxKind.FieldKeyword);
```

Comment:
>AddKeyword(SyntaxKind.FieldKeyword); [](http://example.com/codeflow?start=16&length=36)

We probably don't want to do this when a metadata name is requested

Reply:
The closest flag I found was `SymbolDisplayCompilerInternalOptions.UseMetadataMethodNames`, was this the one you had in mind?

Reply:
Yes something like this. I guess, we might want to make it more general, or add a similar flag for field names and specify it for existing formats that are using ```UseMetadataMethodNames``` today.

Reply:
For testing purposes I am going to try checking and using this flag. I expect this will fix a large number of failing tests, which had started using the new symbol display for backing field.

After I see results of that, I will try to figure out a more permanent solution, perhaps by renaming the flag to `UseMetadataMemberNames`.

I also noticed that this flag is not respected when displaying a property accessor (e.g. `Type.Prop.get` is still used when the flag is set). I added a test to show that. I'm not sure whether we want to change the behavior for that case.